frontend/src/component/ui/Toggle.scss

```diff
@@ -0,0 +1,44 @@
+@import '../styles/corner';
+@import '../styles/color';
+@import '../styles/font';
+
+$toggle-width: 100px / 1.5;
+$toggle-height: 50px / 1.5;
+
+$toggle-radius: 80px;
+$toggle-shrink-size: 4px;
+
+.toggle-component {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+
+  .toggle-back {
+    display: inline-block;
+    background-color: lightgray;
+    width: $toggle-width;
+    height: $toggle-height;
+    border-radius: $toggle-radius;
+    cursor: pointer;
+    transition: background-color 0.3s;
+  }
+
+  .toggle-back.active {
```

Comment:
```suggestion
  . background.active {
```

Reply:
I think the space in front of "background" shouldn't be there since it's a class. I'll remove it.